plugins/modules/zos_apf.py

```diff
@@ -451,6 +461,7 @@ def main():
                 sms=dict(arg_type='bool', required=False, default=False),
             )
         ),
+        tmphlq=dict(type='qualifier_or_empty', required=False, default=""),
```

Comment:
Why is this defaulting to blank/empty?

Reply:
Resolving because I created issue #387 - it was a good comment and point, please review the issue as well @fernandofloresg